package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-chromium",
-  "version": "5.0.0",
+  "version": "6.0.0",
```

Comment:
@jensengar went ahead and bumped this in the branch so it's right in master upon merge.

This definitely ended up as a breaking change so this is the major is definitely the right level to bump